hack/test/e2e-qemu.sh

```diff
@@ -100,7 +100,7 @@ case "${WITH_DISK_ENCRYPTION:-false}" in
   false)
     ;;
   *)
-    QEMU_FLAGS+=("--encrypt-ephemeral" "--encrypt-state")
+    QEMU_FLAGS+=("--encrypt-ephemeral" "--encrypt-state", "--disk-encryption-key-types", "kms")
```

Comment:
don;t we need a different test for this, one that tests with `uuid` and another that tests kms?

Reply:
I've changed `apply-config.go` to add `nodeID` key type too, so it should be fine now.